setup.py

```diff
@@ -3,8 +3,8 @@
 from distutils.core import setup
 
 setup(name='AutoCertKit',
-      version='1.2.0',
+      version='@KIT_VERSION@',
       author='Citrix System Inc.',
       url='http://github.com/xenserver/auto-cert-kit',
-      packages=['autocertkit', 'XenAPI', 'acktools', 'acktools.net'],
+      packages=['acktools', 'acktools.net'],
```

Comment:
Actually setup.py was not used anywhere at all. Instead mk/setup.py was being used by rpm to install acktools and acktool.net. whereas we place XenAPI file directly into desired path and dont have any package registration done across it.

I have removed setup.py and moved /mk/setup.py as /setup.py

We should actually do apropriate package setup for XenAPI as well. I will keep this is as future work.